src/renderer/actions/install-path.utils.tsx

```diff
@@ -1,7 +1,76 @@
-import settings, { msStoreBasePath, steamBasePath } from 'renderer/rendererSettings';
+import * as fs from 'fs';
+import walk from 'walkdir';
+import * as path from 'path';
+import * as os from 'os';
+import settings from 'renderer/rendererSettings';
 import { Directories } from 'renderer/utils/Directories';
 import { dialog } from '@electron/remote';
-import fs from 'fs';
+import { managedSim, TypeOfSimulator } from 'renderer/utils/SimManager';
+
+const possibleBasePaths = {
+  msfs2020: {
+    store: path.join(Directories.localAppData(), '\\Packages\\Microsoft.FlightSimulator_8wekyb3d8bbwe\\LocalCache\\'),
+    steam: path.join(Directories.appData(), '\\Microsoft Flight Simulator\\'),
+  },
+  msfs2024: {
+    store: path.join(Directories.localAppData(), '\\Packages\\Microsoft.Limitless_8wekyb3d8bbwe\\LocalCache\\'),
+    steam: path.join(Directories.appData(), '\\Microsoft Flight Simulator 2024\\'),
+  },
+};
+
+export const msfsBasePath = (sim: TypeOfSimulator): string => {
+  if (os.platform().toString() === 'linux') {
+    return 'linux';
+  }
+
+  // Ensure proper functionality in main- and renderer-process
+  let msfsConfigPath = null;
+
+  const steamPath = path.join(possibleBasePaths[sim].steam, 'UserCfg.opt');
+  const storePath = path.join(possibleBasePaths[sim].store, 'UserCfg.opt');
+  if (fs.existsSync(steamPath) && fs.existsSync(storePath)) return 'C:\\';
+  if (fs.existsSync(steamPath)) {
+    msfsConfigPath = steamPath;
+  } else if (fs.existsSync(storePath)) {
+    msfsConfigPath = storePath;
+  } else {
+    walk(Directories.localAppData(), (path) => {
+      if (path.includes('Flight') && path.includes('UserCfg.opt')) {
+        msfsConfigPath = path;
+      }
+    });
+  }
+
+  if (!msfsConfigPath) {
+    return 'C:\\';
```

Comment:
If we can't find something we should return `null` I think, not some special values.

Reply:
This is because a non-existing path used to break something. Not sure if this is still the case, I copied this over. Can investigate maybe